lesson 1/README.md

```diff
@@ -26,17 +26,54 @@ Use this to properly scope your project and to make sure that your team is on tr
 3. Peers can (and should) checkout the branch to test it out before merging.
 
 #### 5. Merge Conflicts:
-Working in a relatively large project with multiple other people will result in merge conflicts. It happens when the same line of code has been changed on master in your branch. Merge conflicts might look something like this:
+Working in a relatively large project with multiple other people will result in merge conflicts. It happens when the same line of code has been changed on master in your branch. Here's an example of a pull request with merge conflicts:
+
+![conflict](images/conflict.png)
+
+Let's resolve the merge conflict my using `rebase`!
+
```

Comment:
ah woops i meant to change that. I thought I did. My bad. I decided teaching them to rebase was too much for the first class.